instrumentation/grape/lib/opentelemetry/instrumentation/grape/subscriber.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Grape
+      # Manages all subscriptions, both for custom subscribers and built-in notifications
+      class Subscriber
+        class << self
+          # Subscribe to all custom and built-in notifications (except those specified in the :ignored_events configs)
+          def subscribe
+            subscribe_to_custom_notifications
+            subscribe_to_built_in_notifications
+          end
+
+          private
+
+          SUBSCRIPTIONS = {
```

Comment:
Proposal. Feel free to decline if this is too verbose etc.

```suggestion
          # ActiveSupport::Notifications that can be subscribed to using the 
          # documented `.subscribe` interface.
          # 
          # Reference: https://api.rubyonrails.org/classes/ActiveSupport/Notifications.html#method-c-subscribe
          SUBSCRIPTIONS = {
```

Reply:
Added this 😄